test/integration/auth.spec.ts

```diff
@@ -75,10 +76,18 @@ const mockUserData = {
   photoURL: 'http://www.example.com/' + newUserUid + '/photo.png',
   disabled: false,
 };
-const actionCodeSettings = {
+const actionCodeSettings : ActionCodeSettings = {
   url: 'http://localhost/?a=1&b=2#c=3',
   handleCodeInApp: false,
 };
+const actionCodeSettingsWithCustomDomain: ActionCodeSettings = {
+  url: 'http://localhost/?a=1&b=2#c=3',
+  handleCodeInApp: true,
+  linkDomain: 'liubin-dev2.testdomaindonotuse.com',
```

Comment:
Please update these to use the CI project's linkDomain and bundle-id

Reply:
cc @lahirumaramba do we have an iOS app setup in the CI test project?

Reply:
Looks like bundle-id is not necessary, can you please share the linkDomain for the CI project, will update?